docs/concepts/resources/revisions.md

```diff
@@ -0,0 +1,29 @@
+# Revisions
+
+--8<-- "about-revisions.md"
+
+## Related concepts
+
+* **[Autoscaling](../../serving/autoscaling/README.md){target=_blank}**
+    - Revisions can be automatically scaled up and down according to incoming traffic.
+
+* **[Configuring gradual rollout of traffic to Revisions](../../serving/rolling-out-latest-revision.md){target=_blank}**
+    - Revisions enable progressive roll-out and roll-back of application changes by changing the HTTP routing between Service names and Revision instances.
```

Comment:
I think routing for a specific service can be changed between different revisions, but the wording `Service names and Revision instances` implied to me that you can route to a different service

Reply:
I think this can be simplified actually to just the first part of the sentence. Thanks!